src/pages/local-grants/colombia/es.md

```diff
@@ -0,0 +1,89 @@
+
+### **¿Quién es elegible para participar?**
+
+- Cualquier persona interesada en contribuir al fortalecimiento de la comunidad Ethereum Colombia.
+
+### ¿Cómo puedo aplicar a una subvención local?
+
+- Solo debes llenar [este formulario](/es/local-grants/colombia/#submit).
+- Nota: no existe un límite para el numero de aplicaciones
+  - ¡Esto quiere decir que estaremos felices de apoyar más de una idea o proyecto!
+
+### ¿Qué tipos de proyectos son considerados de interés?
+
+- Cualquier proyecto (técnico, educativo, comunitario y/o cualquier otro), que beneficie el ecosistema Ethereum Colombiano.
+- ¡Sorpréndenos con tu creatividad! Acá va una lista de preguntas que te puede servir como fuente de inspiración:
+  - ¿Estás construyendo un proyecto técnico en Ethereum?
+  - ¿Estás construyendo infraestructura pública o alguna herramienta de fuente abierta (open source) para los desarrolladores de Ethereum?
+  - ¿Estás construyendo alguna dApp en Ethereum, que responda a una necesidad clara de tu comunidad o del ecosistema Ethereum?
+  - ¿Estas trabajando en Layer 2, Eth2, o algún reto de interoperabilidad?
+  - ¿Estas construyendo recursos comunitarios o educativos, de calidad, en español o en lenguas indígenas locales?
+  - ¿Estás construyendo puentes de comunicación con universidades, compañías locales entre otros actores locales, con el objetivo de crecer y apoyar el fortalecimiento y crecimiento de la comunidad Ethereum?
+  - ¿Estás trabajando en proyectos que faciliten la incorporación de usuarios en tu comunidad a través de dApps, DAO’s, o la familia Ethereum extendida?
+  - ¿Estás contribuyendo al crecimiento de la base de desarrolladores Ethereum?
+  - ¿Estás trabajando en alguna investigación relacionada con Ethereum (criptografía, Eth2, sistema de pruebas)?
+- ¿Necesitas inspirarte un poco más? Consulta esta [lista de deseos](/en/wishlist/) general.
+
+### **¿Cuáles son los criterios de evaluación?**
+
+- Factibilidad.
+- Componente innovador.
+- Impacto potencial.
+- Usabilidad.
+- Sostenibilidad del proyecto.
+- Espíritu de mejora del ecosistema.
+- Alineación con los valores de Ethereum
+
+### **¿Cuáles son los requisitos para presentar una propuesta?**
+
+- Las propuestas deben ser presentadas en inglés.
+- Deben estar relacionadas con Ethereum.
+- Para el caso de propuestas técnicas, el software debe estar en fuente abierta (open source).
+
+### **¿Cuál es la fecha límite de aplicación?**
+
+- Diciembre 17 de 2020.
+
+### **¿Cuál es el valor de las subvenciones locales?**
+
+- Los montos son decididos caso por caso.
+- Los beneficiarios tendrán la opción de recibir el valor en ETH, Dai, o Dólares Americanos (USD).
+
+## **🚀 ¿Cómo aplicar?** {#submit}
+
+_Completa el siguiente formulario de inscripción, para dejarnos saber sobre tu ideas relacionadas con Ethereum o tu proyecto ya existente; eso es todo. ¡Tras recibir tu aplicación, nos podremos en contacto vía correo electrónico!._
+
+<LocalGrantsForm wave="Colombia | 2020" />
```

Comment:
@gilbertginsberg note: this form is still in English. I'll need all the translated prompts / field names if we want to translate that as well.

Reply:
I'll work on this. But we can deploy with the English version and then update later.